docs/extras.rst

```diff
@@ -256,23 +262,32 @@ diffs:
         <tbody>
           {% for item in object_history %}
             <tr>
-              <th scope="row">{{ item.pgh_created_at|date:"DATETIME_FORMAT" }}</th>
-              <td>{{ item.pgh_label }}</td>
-              <td align="right">
+              <th scope="row" style="width: 20%;">{{ item.pgh_created_at|date:"DATETIME_FORMAT" }}
+                {% if item.pgh_context%}
+                  <br>
+                    {% if item.pgh_context.metadata.user %}
+                      by user {{item.pgh_context.metadata.user}}
+                    {% elif item.pgh_context.metadata.url and "login" in item.pgh_context.metadata.url %}
+                      login
```

Comment:
Is this something specific you are tracking in your app? It looks like it is divided between login requests and requests where there's an authenticated user.

I'd prefer if the example in the docs is generic. Mind if I remove this `if` block?

Reply:
Hi thanks for the commenting -- and yes, please remove the block if needed, as long as we can see who is doing the changing. Thanks!